idd/Energy+.idd.in

```diff
@@ -24527,10 +24527,57 @@ ZoneEarthtube,
       \note  "C" in Equation
       \type real
       \default 0
- N18; \field Velocity Squared Term Flow Coefficient
+ N18, \field Velocity Squared Term Flow Coefficient
       \note  "D" in Equation
       \type real
       \default 0
+ A5,  \field Earth Tube Model Type
```

Comment:
Yay for no transition!  Right?

Reply:
Yes, correct.  Intentionally done that way.  The old model is still the default (notice that the existing earth tube idf in the test suite did not get changed and it runs with no diffs).